ui/imports/shared/controls/TokenDelegate.qml

```diff
@@ -96,7 +96,7 @@ StatusListItem {
             StatusTextWithLoadingState   {
                 id: currencyBalance
                 anchors.right: parent.right
-                text: modelData ? LocaleUtils.currencyAmountToLocaleString(modelData.enabledNetworkCurrencyBalance) : ""
```

Comment:
Where has this value gone, what's the text now when it's not loading?

Reply:
its assigned from the outside now in AssetsView.qml